src/AlertsManagement/AlertsManagement/help/Update-AzSmartGroupState.md

```diff
@@ -0,0 +1,123 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.AlertsManagement.dll-Help.xml
+Module Name: Az.AlertsManagement
+online version:
+schema: 2.0.0
+---
+
+# Update-AzSmartGroupState
+
+## SYNOPSIS
+{{ Fill in the Synopsis }}
+
+## SYNTAX
+
+```
+Update-AzSmartGroupState -SmartGroupId <String> -State <String> [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+{{ Fill in the Description }}
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> {{ Add example code here }}
+```
+
+{{ Add example description here }}
```

Comment:
need to remove all these instances of `{{ }}`

Reply:
Updated all help generation files as suggested.

Reply:
Yes, set the `online version: ` field at the top of the file to use: `https://docs.microsoft.com/en-us/powershell/module/<module-name-lowercase>/<cmdlet-name-lowercase>`

Reply:
This is what I have done - https://github.com/Azure/azure-powershell/pull/9495/files#diff-1a8b1dbc1919c56e6f260e49d5cb3518
Still it is failing.